Description: Removes google contacts plugin, from building as currently there is no easy way to compile gdata lib.
Index: jitsi/build.xml
===================================================================
--- jitsi.orig/build.xml
+++ jitsi/build.xml
@@ -1047,7 +1047,7 @@
         bundle-netaddr,bundle-netaddr-slick,bundle-slickless,
         bundle-slick-runner,bundle-sip,bundle-sip-slick,bundle-fileaccess,
         bundle-fileaccess-slick,bundle-neomedia,bundle-ldap,
-        bundle-googlecontacts-service,bundle-googlecontacts,
+        bundle-googlecontacts-service,
         bundle-hid-service,bundle-hid,
         bundle-resource-manager,bundle-resources-defaultpack,
         bundle-protocol,bundle-protocol-media,bundle-icq,
Index: jitsi/lib/felix.client.run.properties
===================================================================
--- jitsi.orig/lib/felix.client.run.properties
+++ jitsi/lib/felix.client.run.properties
@@ -98,7 +98,6 @@ felix.auto.start.45= \
  reference:file:sc-bundles/provdisc-dhcp.jar \
  reference:file:sc-bundles/provdisc-mdns.jar \
  reference:file:sc-bundles/googlecontacts-service.jar \
- reference:file:sc-bundles/googlecontacts.jar \
  reference:file:sc-bundles/certificate.jar
 
 felix.auto.start.49= \
